internal/Employee.go

```diff
@@ -1,6 +1,16 @@
 package internal
 
-type Employee struct {
+type Employee interface{}
+
+type EmployeeResponse struct {
+	Employee
```

Comment:
I didn't get the purpose why do you declare and then embed the empty interface?

Reply:
+